app/assets/javascript/pageflow/progress_navigation_bar/widget.js

```diff
@@ -157,8 +157,24 @@
         e.preventDefault();
       }
 
+      function toggleChapterDisplay() {
+        var dots = $('.navigation_dots a'),
+            chaptersInCurrentStoryLine;
+
+        chaptersInCurrentStoryLine = _.uniq(dots.filter(':not(.filtered)').get().map(function (pageLink) {
+          return $(pageLink).data('chapterId');
+        }));
+
+        if (chaptersInCurrentStoryLine.length == 1) {
+          $('.navigation_site_detail .chapter').hide();
+        } else {
+          $('.navigation_site_detail .chapter').show();
```

Comment:
Unscoped jQuery selection might lead to elements outside of navigation bar to be affected. Either pass `that.element` as second parameter or use `that.element.find`. Same for `dots` above.